doc/object.types.property.md

```diff
@@ -0,0 +1,153 @@
+@typedef {PrimitiveValue|PrimitiveConstructor|Constructor|TypeObject|Function|PropDefinition} can-define-object/object.types.property Property
```

Comment:
Seems like maybe this isn't working:
![image](https://user-images.githubusercontent.com/5851984/58817622-c9c11e80-85f1-11e9-8daa-33392d7c819c.png)

Reply:
Yeah, it's not, I spent a bunch of time trying to figure out why. Haven't figured it out yet.